seeds.rb

```diff
@@ -3,16 +3,16 @@ def random_time
 end
 
 tasks = [
-  { name: "The First Task", description: "", completed_at: random_time },
-  { name: "Go to Brunch", description: "" },
-  { name: "Go to Lunch", description: "", completed_at: random_time },
-  { name: "Go to Second Lunch", description: "" },
-  { name: "Play Video Games", description: "", completed_at: random_time },
-  { name: "High Five Somebody You Don't Know", description: "", completed_at: random_time },
-  { name: "Plant Flowers", description: "", completed_at: random_time },
-  { name: "Call Mom", description: "" },
-  { name: "She worries, you know.", description: "" },
-  { name: "Nap.", description: "", completed_at: random_time }
+  { task: "The First Task", comments: "", date_completed: random_time },
+  { task: "Go to Brunch", comments: "" },
+  { task: "Go to Lunch", comments: "", date_completed: random_time },
+  { task: "Go to Second Lunch", comments: "" },
+  { task: "Play Video Games", comments: "", date_completed: random_time },
+  { task: "High Five Somebody You Don't Know", comments: "", date_completed: random_time },
+  { task: "Plant Flowers", comments: "", date_completed: random_time },
+  { task: "Call Mom", comments: "" },
+  { task: "She worries, you know.", comments: "" },
+  { task: "Nap.", comments: "", date_completed: random_time }
 ]
```

Comment:
When you're creating seeds you don't (and shouldn't have to) put in blank strings for columns that you don't have data for.